src/constants/stake.json

```diff
@@ -1,5 +1,27 @@
 {
   "syrup": [
+    {
+      "token": "TOWER",
```

Comment:
Was this included on masters, if so remove the code and then we'll make a seperate PR to rebase dev with master.

Reply:
Removed these updates